src/gc-debug.c

```diff
@@ -1,6 +1,7 @@
 // This file is a part of Julia. License is MIT: https://julialang.org/license
 
-#include "gc.h"
+#include "gc-common.h"
+#include "gc-stock.h"
```

Comment:
This can be done later, but maybe we should have a `gc.h` which conditionally includes `gc-stock.h` or `gc-mmtk.h`. Then we only have to include `gc.h` where needed.

If you did this now, you won't have to go back and change these files.

Reply:
Yes, let's leave to a subsequent PR.